module/data/other/roll-options.mjs

```diff
@@ -217,11 +220,11 @@ export default class EdRollOptions extends foundry.abstract.DataModel {
       available:  actor.system.devotion.value,
       step:       actor.system.devotion.step,
     };
-    const karma = { pointsUsed: actor.system.karma.useAlways ? 1 : 0,
-      available:  actor.system.karma.value, 
-      step:       actor.system.karma.step 
+    data.karma = {
+      pointsUsed: actor.system.karma.useAlways ? 1 : 0,
+      available:  actor.system.karma.value,
+      step:       actor.system.karma.step
     };
-    data.karma = karma;
     data.devotion = devotion;
```

Comment:
this should be removed as well and data.devotion should replace const devotion above right

Reply:
@ChristopherSD